test/checkPath.js

```diff
@@ -8,11 +8,9 @@ const chalk = require('chalk');
 
 (async () => {
     if (packagePath.length > 0) {
-        const checkPath = await checkPathCorrect(packagePath);
+        const checkPath = isPathCorrect(packagePath);

```

Comment:
Is it async or sync? I am asking to make sure this method is sync because the previous one is async and you used `await`

Reply:
It's sync. The method was previously async, because of the spinner, but I later decided since the spinner wasn't working at all to change it to be sync :D